mlflow/projects/__init__.py

```diff
@@ -194,7 +203,7 @@ def _run(uri, experiment_id, entry_point="main", version=None, parameters=None,
 def run(uri, entry_point="main", version=None, parameters=None,
         docker_args=None, experiment_name=None, experiment_id=None,
         backend=None, backend_config=None, use_conda=True,
-        storage_dir=None, synchronous=True, run_id=None):
+        storage_dir=None, synchronous=True, run_id=None, run_name=None):
```

Comment:
@AgrawalAmey Can we add an entry to the method docstring for the `run_name` parameter?